serverless/develop/serverless-autoscaling-developer.adoc

```diff
@@ -1,17 +1,29 @@
-[id="serverless-autoscaling-scale-bounds"]
-= Scale bounds
+[id="serverless-autoscaling-developer"]
+= Autoscaling
 include::modules/common-attributes.adoc[]
 include::modules/serverless-document-attributes.adoc[]
-:context: serverless-autoscaling-scale-bounds
+:context: serverless-autoscaling-developer
 
 toc::[]
 
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. For example, if an application is receiving no traffic, and scale to zero is enabled, Knative Serving scales the application down to zero replicas. If scaling to zero is disabled, the application is scaled down to the minimum number of replicas specified for applications on the cluster. Replicas can also be scaled up to meet demand if traffic to the application increases.
```

Comment:
> scaled down to the minimum number of replicas specified for applications on the cluster.
Have a pointer for that?

Reply:
Yeah, we have minScale docs so I can add a link for that :)